base/replutil.jl

```diff
@@ -444,6 +456,10 @@ function show_method_candidates(io::IO, ex::MethodError, kwargs::Vector=Any[])
             buf = IOBuffer()
             tv = Any[]
             sig0 = method.sig
+            # @Hardcoded: don't display the sysimg.jl method
```

Comment:
This convention isn't used anywhere in the code base currently, so please don't introduce it in this PR. If we really want to use this kind of convention, it needs to be discussed and used by everybody. Else we'll end up with as many conventions as contributors.